Lib/fontbakery/profiles/googlefonts.py

```diff
@@ -1084,7 +1084,7 @@ def com_google_fonts_check_name_version_format(ttFont):
   from fontbakery.utils import get_name_entry_strings
   import re
   def is_valid_version_format(value):
-    return re.match(r'Version\s0*[1-9]+\.\d+', value)
+    return re.match(r'Version\s0*[1-9][0-9]*\.\d+', value)
```

Comment:
do we really need this change? Arent the two regexes equivalent? `+` means "one or more".

Reply:
oh! Now I see the difference! Yes, you are right!